### ISSUE TITLE
Internal server error when requesting a carpool

### ISSUE THREAD

**Issue by jillh510** [iandees/carpools#35]
Steps to reproduce
Go to "Look for a ride"
Click on a carpool
Click on "Request a seat"
Fill in my gender
Click "Request a ride"
Result: 'Internal Server Error'

**Comment by iandees** [comment]
```
2017-04-17T21:42:51.649246+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/werkzeug/routing.py", line 1768, in build
2017-04-17T21:42:51.649253+00:00 app[web.1]: BuildError: Could not build url for endpoint 'carpool.cancel_rider' with values ['carpool_id']. Did you mean 'carpool.new_rider' instead?
```

**Comment by iandees** [comment]
This should be fixed in a14de60.
